src/EncompassRest/Loans/UpdateLoanOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using EncompassRest.Utilities;
+
+namespace EncompassRest.Loans
+{
+    public sealed class UpdateLoanOptions : LoanOptions
+    {
+        public bool AppendData { get; set; }
```

Comment:
we should have defaults for this.

Reply:
Agreed, I think this one might ought to be `true` so as not to overwrite existing data. Thoughts?